dace/frontend/python/replacements.py

```diff
@@ -600,6 +609,16 @@ def _log(pv: 'ProgramVisitor', sdfg: SDFG, state: SDFGState, input: str):
     return _simple_call(sdfg, state, input, 'log')
 
 
+@oprepo.replaces('math.floor')
+def _floor(pv: 'ProgramVisitor', sdfg: SDFG, state: SDFGState, input: str):
```

Comment:
I think we need special handling for those. `_simple_call` is going to create output that has the same dtype as the input. However, I quickly checked with Python 3.8 and this is not correct for floor and ceil:
```
Python 3.8.5 (default, Sep  4 2020, 07:30:14) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from math import floor, ceil
>>> floor(0.5)
0
>>> type(floor(0.5))
<class 'int'>
>>> type(ceil(0.5))
<class 'int'>
>>> 
```

Reply:
Fixed and pushed